cxx/util_distribution_variant.cc

```diff
@@ -5,13 +5,7 @@
 
 #include <cassert>
 #include <sstream>
-
-#include "distributions/beta_bernoulli.hh"
```

Comment:
I looked this up the other day and the style guide says to include these: https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.md?cl=head#Include_What_You_Use (IMO we might as well follow that but I don't feel strongly)

Reply:
Done.